enclave_manager/avalon_enclave_manager/wpe/wpe_requester.py

```diff
@@ -117,7 +119,7 @@ def get_unique_verification_key(self, verification_key_nonce):
 
     def register_wo_processor(self, unique_verification_id,
                               encryption_key,
-                              proof_data):
+                              proof_data, mr_enclave):
```

Comment:
Add mr_enclave to the parameter list, around line 130.

Reply:
Done.